google/cloud/spanner_v1/_helpers.py

```diff
@@ -87,6 +97,27 @@ def _merge_query_options(base, merge):
     return combined
 
 
+def _assert_numeric_precision_and_scale(value):
+    """
+    Spanner supports fixed 38 digits of precision and 9 digits of scale.
+    This number can be optionally prefixed with a plus or minus sign.
+    Read more here: https://cloud.google.com/spanner/docs/data-types#numeric_type
+
+    Asserts that input numeric field is within spanner supported range.
```

Comment:
```suggestion
    Asserts that input numeric field is within Spanner supported range.
```
Also this should be moved to the top of the docstring. That way you have an explanation of what function does, followed by why it's needed.

Reply:
Done.